### ISSUE TITLE
Question: How top verify ShaSum of install script at https://get.sdkman.io

### ISSUE THREAD

**Issue by msgilligan** [sdkman/sdkman-cli#1040]
**Question**
It would be nice if there were a simple and documented mechanism for verify the Sha256Sum of the install script at https://get.sdkman.io. I searched around a little (see below) and didn't find anything.

Is there a simple mechanism? If so where is it documented?

Something along the lines of:

```
curl -s "https://get.sdkman.io" > install_sdkman.sh    
sha256sum install_sdkman.sh
# compare to ??
```

My homework:

- [x] I have checked [the documentation](https://sdkman.io/usage)
- [x] I have done a quick search in [past issues](https://github.com/rclone/rclone/issues?q=) or [Stack Overflow](https://stackoverflow.com/questions/tagged/sdkman) for similar problems
- [ ] I have brought up a conversation in the [Slack User Issues Channel](https://sdkman.slack.com/app_redirect?channel=user-issues)

**Comment by msgilligan** [comment]
> Wouldn't it make more sense to get a checksum on the SDKMAN zip dist instead? 

Yes. That sounds right to me. Is this documented somewhere? Did I miss it?

**Comment by marc0der** [comment]
This doesn't exist yet, but given that we use jreleaser and that SDKMAN is now itself an SDK, it makes great sense. Happy for you to change this into a feature request.

**Comment by msgilligan** [comment]
I changed the title is there anything else I need to do?

**Comment by aalmiray** [comment]
@marc0der currently the checksum resides inside a file named `checksums_sha256.txt`. This file collects checksums for all assets. Right now there's just one artifact. Another option is to publish checksums for individual files such that `sdkman-cli-5.13.1.zip` gets its checksum published as `sdkman-cli-5.13.1.zip.sha256`. JReleaser can do that of course.
